Lib/test/test_xml_etree.py

```diff
@@ -122,6 +128,104 @@ def test_all(self):
         support.check__all__(self, ET, names, blacklist=("HTML_EMPTY",))
 
 
+class ElementTree_Element_UnitTest(unittest.TestCase):
+
+    def test___init__(self):
+        tag = "foo"
+        attrib = { "zix": "wyp" }
+
+        element_foo = ET.Element(tag, attrib)
+
+        with self.subTest("traits of an element"):
+            self.assertIsInstance(element_foo, ET.Element)
+            self.assertIn("tag", dir(element_foo))
+            self.assertIn("attrib", dir(element_foo))
+            self.assertIn("text", dir(element_foo))
+            self.assertIn("tail", dir(element_foo))
+
+        with self.subTest("string attributes have expected values"):
+            self.assertEqual(element_foo.tag, tag)
+            self.assertIsNone(element_foo.text)
+            self.assertIsNone(element_foo.tail)
+
+        with self.subTest("attrib is a copy"):
+            self.assertIsNot(element_foo.attrib, attrib)
+            self.assertEqual(element_foo.attrib, attrib)
+
+        with self.subTest("attrib isn't linked"):
+            attrib["bar"] = "baz"
+
+            self.assertIsNot(element_foo.attrib, attrib)
+            self.assertNotEqual(element_foo.attrib, attrib)
+
+
+    def test___copy__(self):
+        element_foo = ET.Element("foo", { "zix": "wyp" })
+        element_foo.append(ET.Element("bar", { "baz": "qix" }))
+
+        element_foo2 = copy.copy(element_foo)
+
+        with self.subTest("elements are not the same"):
```

Comment:
I think a subtest is too much overhead for something _this_ simple. Add a comment if you want, but not a subtest.

Reply:
subTest() is for use in loops, where we can not identify the failed case by traceback.

Reply:
I think it's also useful when you use a parent test function as simply a scope to define common elements used by multiple tests, but ok.